backend/src/v5/routes/teamspaces/projects/models/common/tickets.groups.js

```diff
@@ -44,8 +44,10 @@ const getGroup = (isFed) => async (req, res, next) => {
 	const { teamspace, project, model, ticket, group } = params;
 
 	try {
+		const convertTo3DRepoGuids = query.convertIds !== 'false';
```

Comment:
rename `convertTo3DRepoGuids` to something similar in the ticket groups processor suggestion

Reply:
Also missing swagger update for this additional flag